### PR TITLE
Added a lot of bot lists

### PR DESCRIPTION
**Pull request by GregTCLTK**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -130,6 +130,11 @@
 - [discordbots.org](https://discordbots.org/)
 - [bots.discord.pw](https://bots.discord.pw/)
 - [Carbonitex](https://carbonitex.net/discord/bots)
+- [botlist.space](https://botlist.space)
+- [Bots For Discord](https://botsfordiscord.com)
+- [Boat-List](https://boat-list.glitch.me)
+- [Discord Bot World](https://discordbot.world)
+- [ls.terminal.link](https://ls.terminl.link)
```
Comment (byparallax) [inline readme.md]: Seems to be down.

File: readme.md [hunk 2]
```diff
@@ -130,6 +130,11 @@
 - [discordbots.org](https://discordbots.org/)
 - [bots.discord.pw](https://bots.discord.pw/)
 - [Carbonitex](https://carbonitex.net/discord/bots)
+- [botlist.space](https://botlist.space)
+- [Bots For Discord](https://botsfordiscord.com)
+- [Boat-List](https://boat-list.glitch.me)
```
Comment (byparallax) [inline readme.md]: Hosted on Glitch, therefore unreliable.

File: readme.md (hunk 1 above)
Comment (GregTCLTK) [reply readme.md]: Which

Comment (OvermindDL1) [reply readme.md]: > [ls.terminal.link](https://ls.terminl.link)

File: readme.md [hunk 3]
```diff
@@ -134,7 +134,7 @@
 - [Bots For Discord](https://botsfordiscord.com)
 - [Boat-List](https://boat-list.glitch.me)
 - [Discord Bot World](https://discordbot.world)
-- [ls.terminal.link](https://ls.terminl.link)
+- [ls.terminal.link](https://ls.terminl.ink)
```
Comment (OvermindDL1) [inline readme.md]: > https://ls.terminl.ink

This has no DNS entry either?

File: readme.md (hunk 1 above)
Comment (GregTCLTK) [reply readme.md]: Its was https://ls.terminl.ink :D

**Pull request closed without merging**